### ISSUE TITLE
cosmetical request: vst volume control

### ISSUE THREAD

**Issue by careholding** [kmatheussen/radium#1114]
Not super important, Just an idea;
Sometimes (especially when making string or drone sounds) I like to control to volume of a note from a vst instrument after the first midimessage has been send to the vst..

I can do that by making an automation on system volume, so this request is not very important and maybe also hard to implement;
But it would be nice if I somehow could choose to use the velocity bar for this, simply because it reduces clutter and the velocity bar gives much better visibility/ease of use..

**Comment by kmatheussen** [comment]
I don't think it's hard to implement, but it sounds messy. Are you thinking of an option to let note velocity control the volume of an instrument?

**Comment by careholding** [comment]
yes exactely, to get something like the volume control of sampleplayer. But yeah maybe messy, and automation works fine.. although sampleplayer looks alot nicer

sampleplayer:
![git1](https://user-images.githubusercontent.com/19477905/38816665-e11615a4-4196-11e8-892f-23f555996e65.JPG)

vst:
![git2](https://user-images.githubusercontent.com/19477905/38816666-e13f3420-4196-11e8-8b3e-9689d1570b04.JPG)
